src/service/apply/applyBondUser.ts

```
@@ -0,0 +1,61 @@
+import { AuthenticatedRequest, BasicResponse } from '../../types';
+import { Response } from 'express';
+import { ApplyBondUserData, ApplyUserQuery, ApplyBondUserResponse } from '../../types/apply';
+import { prisma } from '../../config/prisma';
+
+const PAGE_SIZE = 10;
+const DEFAULT_USER_NAME = '무명';
+
+export const applyBondUser = async (req: AuthenticatedRequest<{}, ApplyUserQuery, {}>, res: Response<ApplyBondUserResponse | BasicResponse>) => {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)